### PR TITLE
Set up resources for job periodic-ci-image-import-to-build01

### PR DESCRIPTION
**Pull request by hongkailiu**
Found several of its pods on build01 where are OOMKilled.

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

**Comment by alvaroaleman** [comment]
Which container got oom killed, are you sure it was the test container or maybe sidecar? xref https://github.com/openshift/release/pull/7492

/hold

**Comment by hongkailiu** [comment]
> Which container got oom killed, are you sure it was the test container or maybe sidecar? xref #7492
> 
> /hold

It looks quite green now.
https://prow.svc.ci.openshift.org/job-history/origin-ci-test/logs/periodic-ci-image-import-to-build01

/close

**Comment by hongkailiu** [comment]
follow up https://coreos.slack.com/archives/CHY2E1BL4/p1584545361342700?thread_ts=1584543929.337000&cid=CHY2E1BL4

The pods of failing jobs are deleted (we do not know why) and it is hard to tell which container is OOMKilled.

I would like to try if it fixes the flakiness. Will revert otherwise.

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -26,6 +26,13 @@ periodics:
       image: registry.svc.ci.openshift.org/ocp/4.5:cli
       imagePullPolicy: Always
       name: ""
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+        limits:
+          cpu: 1
```
Comment (alvaroaleman) [inline ci-operator/jobs/infra-periodics.yaml]: Please do not set CPU limits, CPU is a compressible resource

**Comment by hongkailiu** [comment]
It looks like it is CPU consuming jobs

https://prometheus-k8s-openshift-monitoring.apps.build01.ci.devcluster.openshift.com/graph?g0.range_input=30m&g0.expr=container_memory_rss%7Bpod%3D%2224ae2633-6934-11ea-81ec-0a58ac1020c1%22%7D&g0.tab=0&g1.range_input=30m&g1.expr=container_cpu_usage_seconds_total%7Bpod%3D%2224ae2633-6934-11ea-81ec-0a58ac1020c1%22%7D&g1.tab=0

Screen Shot 2020-03-18 at 12 31 05 PM

I will keep watching a bit more.

**Pull request merged**